test/lib.test.js

```diff
@@ -71,8 +71,6 @@ suite('lib', () => {
   });
 
   test('findSources', () => {
-    assert.lengthOf(findSources(), 0);
```

Comment:
This throws an error as of `globby@6`, I could see preserving this behavior but I think this will have to be a major version bump anyway. since we have to drop support for older versions of node.

Reply:
I'm not quite sure what this change is but don't have major objections